workflow.py

```diff
@@ -90,3 +198,20 @@ def define_common_variables_after_presel(self, variation):
         #print("CvsL sort CvsL:", self.events["JetsCvsL"][self.events["nJetGood"]>=3].btagDeepFlavCvL)
 
         self.events["dijet_csort"] = get_dijet(self.events.JetsCvsL)
+        
+        # Suppress FutureWarning
```

Comment:
We should have an option in the config file to create the `root` or not. Could you introduce that as well?